kafka-connector/src/main/java/com/google/pubsub/kafka/source/CloudPubSubSourceTask.java

```diff
@@ -246,6 +255,15 @@ private int selectPartition(Object key, Object value) {
     }
   }
 
+  private Long getLongValue(String timestamp){
+    try {
```

Comment:
Add

if(timestamp == null) {
  return null;
}

That way we can avoid having to throw and catch the exception for what I imagine will be the most typical use case.